mico-core/src/main/java/io/github/ust/mico/core/broker/MicoApplicationBroker.java

```diff
@@ -169,7 +210,11 @@ public MicoApplication copyAndUpgradeMicoApplicationByShortNameAndVersion(String
 
     public void addMicoServiceToMicoApplicationByShortNameAndVersion(String applicationShortName, String applicationVersion, String serviceShortName, String serviceVersion)
         throws MicoApplicationNotFoundException, MicoServiceNotFoundException, MicoServiceAlreadyAddedToMicoApplicationException, MicoServiceAddedMoreThanOnceToMicoApplicationException,
-        MicoApplicationIsNotUndeployedException, MicoTopicRoleUsedMultipleTimesException, MicoServiceDeploymentInformationNotFoundException, KubernetesResourceException, MicoApplicationDoesNotIncludeMicoServiceException {
+        MicoApplicationIsNotUndeployedException, MicoTopicRoleUsedMultipleTimesException, MicoServiceDeploymentInformationNotFoundException, KubernetesResourceException, MicoApplicationDoesNotIncludeMicoServiceException, KafkaFaasConnectorNotAllowedHereException {
+
+        if (serviceShortName.equals(KAFKA_FAAS_CONNECTOR_SERVICE_NAME)) {
+            throw new KafkaFaasConnectorNotAllowedHereException();
```

Comment:
Is there a technical reason to not allow the kafka faas connector here or is this just to avoid confusion? (Both would be valid reasons for this but maybe add the reason in a comment.)

Reply:
Added comment:
```java
// KafkaFaasConnector is not allowed here, because it should be handled differently.
// See method `addKafkaFaasConnectorInstanceToMicoApplicationByVersion`
```